lib/lighthouse/letters_generator/service_error.rb

```diff
@@ -30,8 +30,8 @@ def initialize(exception = nil)
           @status ||= exception['status'].to_i
           @title ||= exception['title']
           @message = exception['detail'] || exception['message']
-          @key ||= error_key
         end
+        @key ||= error_key
```

Comment:
I ran into issues while testing my controller code because the key was not defined. Further down in this file, this method requires the key: 
```
      def i18n_key
        @key
      end
```
When this method is called in the base error class:
```
      def i18n_data
        I18n.t(i18n_key)
      end
```
It raises errors.